projects/elements-demo/src/app/shared/shared.module.ts

```diff
@@ -32,7 +32,16 @@ const ANGULAR_MATERIAL_MODULES = [
 
 @NgModule({
   declarations: [],
-  imports: [CommonModule, LazyElementsModule, ...ANGULAR_MATERIAL_MODULES],
+  imports: [
+    CommonModule,
+    LazyElementsModule.forRoot([
```

Comment:
The `SharedModule` can't be for root as `SharedModule` is usually imported also in every lazy loaded feature module... So for root can be only in `AppModule` or in `CoreModule`